server/routerlicious/packages/protocol-base/src/protocol.ts

```diff
@@ -121,9 +121,7 @@ export class ProtocolOpHandler implements IProtocolHandler {
 				break;
 
 			case MessageType.Propose:
-				// This should become unconditional once (Loader LTS) DeltaManager.processInboundMessage() stops parsing content (ADO #12052)
-				// Note: Until that change is made in the loader, this case will never be hit.
-				// Then there will be a long time of needing both cases, until LTS catches up to the change.
+				// TODO: Update callers to stop parsing the contents and do it here unconditionally
```

Comment:
From my understanding this is a server-only implementation of `IProtocolHandler` and has nothing to do with the code I'm removing from `DeltaManager`.  Does that sound right?

FYI @znewton as a person I know on the server team :)

Reply:
This assessment does not look right. Protocol handler is the code that is used both by client & server, to interpret quorum ops.